packages/lib-classifier/dev/components/App/App.js

```diff
@@ -186,7 +194,7 @@ class App extends React.Component {
           <Box as='header' pad='medium' justify='end' gap='medium' direction='row'>
             <label htmlFor="locale">Language</label>
             <select id="locale" defaultValue={locale} onChange={this.selectLocale}>
-              {project?.available_languages?.map(locale => <option key={locale} value={locale}>{localeMenu[locale]}</option>)}
+              {availableLocales.map(locale => <option key={locale} value={locale}>{localeMenu[locale]}</option>)}
```

Comment:
`availableLocales` is created from `project.configuration.languages` but the original code used `project.available_languages`. Is that a mistake?

Reply:
No, I was trying to unify where available locales for the menu are referenced from inside the project snapshot. For instance, app-project's ProjectHeader `useStores()` grabs locales from `project.configuration.languages`:
https://github.com/zooniverse/front-end-monorepo/blob/61d17dca43b9b43e86bcf02a9a7f06ff60638ae2/packages/app-project/src/components/ProjectHeader/hooks/useStores.js#L6-L9

But after looking at the difference between `project.configuration.languages` and `project.available_locales`, it seems that projects _without_ any translations have `['en']` in `project.available_locales` and `project.configuration.languages` is `undefined`.

Do you think I should refactor app-project's `useStores()` to reference `project.available_languages` instead of `project.configuration`? An `available_languages` value would need to be added to the Project store as well.

Reply:
I’m not sure. The project app works as it is, so I’m not sure what the side effects of that change would be.